modules/oncodrivefml/1.0/config/default.yaml

```diff
@@ -0,0 +1,70 @@
+lcr-modules:
+    
+    oncodrivefml:
+        
+        inputs:
+            # Available wildcards: {seq_type} {genome_build}
+            input_maf: "__UPDATE__"
+            subsetting_categories: "__UPDATE__"
+
+        genome_builds: ["__UPDATE__"]
+        # Available regions options: cds, lincrna, 3utr, 5utr, promoter_splice_sites
+        regions: ["__UPDATE__"]
+
+        maf_processing:
+            sample_sets: ["__UPDATE__"]
+            prepare_mafs: "{SCRIPTSDIR}/generate_smg_inputs/1.1/generate_smg_inputs.R"
+            include_non_coding: False
+            blacklists: []
+            blacklist_drop_threshold: 4
+
+        format_fml_input:
+            input_columns: "5,6,11,13,16"
+            additional_commands: ""
+
+        reference_files_directory: "ref/lcr-modules-references/downloads/oncodrive/"
```

Comment:
I wonder if it confilicts with the paths below? If they are used at the same place does this creates the duplication of the common part `downloads/oncodrive/`?

Reply:
The path in `reference_files_directory:` is only specified to be passed to the global variable `BGDATA_LOCAL`:

>  
     params:
        local_path = CFG["reference_files_directory"]
 ...
     shell:
        op.as_one_line("""
        export BGDATA_LOCAL={params.local_path} &&  
>
This is required so that OncodriveFML looks in the "ref/lcr-modules-references/downloads/oncodrive/" directory for the hg19/hg38 genome reference file instead of looking in the directory "~/.bgdata/" (the global variable BGDATA_LOCAL defaults to ~/.bgdata/ if it is not manually set).